api/controllers/webhook.js

```diff
@@ -122,10 +159,23 @@ module.exports = {
         }
       })
       .catch((err) => {
+        logger.error(err);
+        if (err.message) {
+          res.badRequest(err);
+        } else {
+          res.badRequest();
+        }
+      });
+  },
+  organization(req, res) {
+    signWebhookRequest(req)
```

Comment:
I'm assume the logic here was just copied from above, but this probably isn't the way we want to handle the response to github. If we can verify the signature and there is nothing wrong with the request, we should respond with a success (204). It looks like Github stores the response to the webhooks, I don't think we really want to return the model or a stack trace if there is an error, we handle this in other ways...

Reply:
good catch... yes ... copied and reused ... i will address this ... thanks